client/js/components/statement/originalStatementsTable/OriginalStatementsTableItem.vue

```diff
@@ -289,10 +289,6 @@ export default {
           ? this.statement.authorName
           : `${Translator.trans('role.citizen')} (${Translator.trans('anonymous')})`
 
-        if (this.statement.votesNum > 0) {
-          name += `<br>${Translator.trans('voters')}: ${this.statement.votesNum}`
-        }
-
```

Comment:
Currently we can't get the full number of votes, because the votes are in relation to statement, but not to original statements. The only number we can get is the number of anonym votes. We have therefore decided to leave it out. PO knows about it.